src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -124,23 +124,26 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                     ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
                 }
 
-                ColorConsole.WriteLine(
-                    padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                if (currentLeaderboard.Since.HasValue)
+                {
+                    ColorConsole.WriteLine(
+                        padding,
+                        $"{currentLeaderboard.Since?.ToLocalTime().ToString()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime().ToString()}".Gray());
+                }
 
                 var maxMessageLength = 0;
                 var refreshTime = DateTime.UtcNow.AddMilliseconds(refreshInterval);
                 using (var timer = new Timer(c =>
                 {
-                    var timeLeft = new TimeSpan(0, 0, 0, (int)Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
-                    var message = $"\r{padding}{DateTime.UtcNow.ToLocalTime()} · Refreshing in {timeLeft.Humanize()}...";
-                    maxMessageLength = Math.Max(maxMessageLength, message.Length);
-                    ColorConsole.Write(message.PadRight(maxMessageLength).DarkGray());
+                    var timeLeft = new TimeSpan(0, 0, 0, (int) Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
+                    var totalMessage = $"Total {itemsName}: ";
+                    var totalValueMessage = $"{currentLeaderboard?.Count ?? 0:N0}"; 
+                    var refreshingMessage = $" · Refreshing in {timeLeft.Humanize()}...";
+                    maxMessageLength = Math.Max(maxMessageLength, refreshingMessage.Length - totalMessage.Length - totalValueMessage.Length - padding.Length);

```

Comment:
I'm not I understand this. E.g. if `totalMessage` is longer than refreshing message, `maxMessageLength` will always be zero.

I think what we need to do, is to construct the `ColorToken[]` which will be passed to `ColorConsole.Write()`, and then concat them into a string and measure that strings length and store that in `maxMessageLength` (if greater than current max message length). Then, we pass the array to the write method, appending an extra string of spaces with the length equal to the difference in length of the max message length and the current message length.